modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/vulcan/openapi/contributor/APIApplicationOpenApiContributor.java

```diff
@@ -312,6 +310,16 @@ else if (type == APIApplication.Property.Type.TEXT) {
 		return schema;
 	}
 
+	private String _getVersion(APIApplication apiApplication) {
+		String version = apiApplication.getVersion();
+
+		if (Validator.isNull(version)) {
```

Comment:
Should we use the isBlank instead? I think it is much faster and it would fit for our use case :thinking: